tests/sur/quota.js

```diff
@@ -196,27 +235,33 @@ function objectMPU(bucket, key, parts, partSize, callback) {
                 },
                 UploadId: uploadId,
             };
-            return s3Client.completeMultipartUpload(params, next);
+            return s3Client.send(new CompleteMultipartUploadCommand(params))
+                .then(() => next())
+                .catch(next);
         },
     ], err => {
-        if (!err && !s3Config.isQuotaInflightEnabled()) {
-            mockScuba.incrementBytesForBucket(bucket, parts * partSize);
+        if (!s3Config.isQuotaInflightEnabled()) {
+            mockScuba.incrementBytesForBucket(bucket, -(parts * partSize));
```

Comment:
i'm not sure to get why we increment the bytes before the call then we decrement them, is there any issue with the test?